exercise_perseus_render/assets/src/vue/perseus.vue

```diff
@@ -249,6 +252,16 @@
           } else {
             this.message = null;
             if (!check.empty) {
+              if (!check.correct) {
+                this.shake = true;
+                if (!this.throttle) {
+                  this.throttle = true;
+                  setTimeout(() => {
+                    this.shake = false;
+                    this.throttle = false;
+                  }, 1000);
+                }
+              }
```

Comment:
I think this can be done with just a single state variable:

```javascript
if (!this.shake) {
  setTimeout(() => {
    this.shake = false;
  }, 1000);
}
this.shake = true;
```